chart/load-balancer-api/templates/api-config.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: api-config
```

```suggestion
  name: {{ include "common.names.fullname" . }}
```

Should we use the `fullname` here like we do for the other templates?

---

✅